lib/elixir/lib/macro.ex

```diff
@@ -158,6 +156,21 @@ defmodule Macro do
       the file and the line number of the quoted source.
     * `:line` - The line number of the AST node.
 
+  The following metadata keys are enabled by `Code.string_to_quoted/2`:
+
+    * `:column` - the column number of the AST node (when `:columns` is true)
+    * `:do` - contains metadata about the `do` location in a function call with
+      `do/end` blocks (when `:pairing_metadata` is true)
+    * `:end` - contains metadata about the `end` location in a function call with
+      `do/end` blocks (when `:pairing_metadata` is true)
+    * `:closing` - contains metadata about the closing pair, such as a `}`
+      in a tuple or in a map, or such as the closing `)` in a function call
+      with parens (when `:pairing_metadata` is true)
+    * `:eol` - is set to true when the opening pair, such as `{` or `(`, are
+      followed by the end of the line (when `:pairing_metadata` is true)
+    * `:opening` - contains the opening for sigils as binaries (such as `"{"`,
```

Comment:
For their names, `:opening` and `:closing` might seem to be related. Maybe prefix the term like `:sigil_opening`, `:sigil_delimiter`, or `:sigil_opening`?

Reply:
The reason that it is called `opening` is because other constructs may have it, such as heredocs. But they are not exposed yet. But I like `:delimiter` much more, so I will go with that. :+1: